### ISSUE TITLE
Modifier keys get "stuck" when switching input source

### ISSUE THREAD

**Issue by Phillipus** [xanderfrangos/twinkle-tray#864]
v.1.16.2

Thanks for making this handy utility!

- I have two input sources for my Dell U2723QE monitor, a Mac Mini and a Windows box
- I assigned shortcut keys Ctrl+F15 to switch input source - DDC command "0x60 27"
- When switching input from Windows and Mac and back to Windows again the Ctrl key is stuck "on", that is I have to press it again to release it. The same is true for any modifier key such as Shift or Alt.

It's as if the modifier key release event is not picked up by TT when switching so that when switching back to Windows it considers the modifier key to be still down.

**Comment by xanderfrangos** [comment]
Is the keyboard shared through the monitor to the Mac/PC? Also, when the key gets stuck, are other apps affected other than Twinkle Tray?

**Comment by Phillipus** [comment]
> Is the keyboard shared through the monitor to the Mac/PC? Also, when the key gets stuck, are other apps affected other than Twinkle Tray?

Yes, the keyboard is shared. All apps (and the OS) are affected when switching back to Windows.

**Comment by xanderfrangos** [comment]
I'm pretty sure this from Windows losing the keyboard mid-press while the input source is changed, rather than Twinkle Tray itself. You may want to try using the Twinkle Tray CLI for this, rather than a shortcut. It'd be something like:

```
"%LocalAppData%\Programs\twinkle-tray\Twinkle Tray.exe" --MonitorNum=1 --VCP="0x60:27"
```

**Comment by Phillipus** [comment]
> I'm pretty sure this from Windows losing the keyboard mid-press while the input source is changed, rather than Twinkle Tray itself. 

If I use Dell's DDPM app to do the same it's OK (I'd rather use TT). Is TT trapping key presses?

**Comment by brunoosti** [comment]
I have a similar issue: when switching monitors or awakening from sleep state some shortcut key get stuck and i can't access the display (it blinks in and out). If I press any brightness key combo (win ctrl up  / wind ctrl down) it "unlocks" my monitors menu (LG 34GP63A-B and 39GS95QE)
